query.go

```diff
@@ -2090,6 +2090,69 @@ func (q *Query) BufferSizeVar(attributeOrDimension string) (uint64, uint64, erro
 	return offsetNumElements, dataNumElements, nil
 }
 
+// BufferSizeVarNullable returns the size (in num elements) of the backing C buffers for the given variable-length nullable attribute
+func (q *Query) BufferSizeVarNullable(attributeOrDimension string) (uint64, uint64, uint64, error) {
```

Comment:
Since nullable support is only for attributes maybe you need to set attributeName or so as an argument

Reply:
The below code is also wrong because of this. You don't fetch the datatype of the attribute but actually looks like it unset except for `TILEDB_COORS`? That is an impossible condition cause `TILEDB_COORDS` can't be nullable.

Reply:
Good catch, fixed